src/caret_analyze/plot/metrics_base.py

```diff
@@ -55,16 +55,19 @@ def _convert_timeseries_records_to_sim_time(
         timeseries_records_list: list[RecordsInterface]
     ) -> list[RecordsInterface]:
         # get converter
+        records_range = Range([to.to_records() for to in self.target_objects])
```

Comment:
What does the variable name `to` mean?
How about: `obj`

Reply:
I used the code from "timeseries.py" as a reference.